.github/workflows/release.yml

```diff
@@ -11,26 +11,31 @@ on:
 jobs:
   build:
     if: github.repository == 'jazzband/pip-tools'
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
 
-      - name: Cache
-        uses: actions/cache@v2
-        with:
-          path: ~/.cache/pip
-          key: deploy-${{ hashFiles('**/setup.py') }}
-          restore-keys: |
-            deploy-
-
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
           python-version: 3.9
 
+      - name: Get pip cache dir
+        id: pip-cache
+        run: |
+          echo "::set-output name=dir::$(pip cache dir)"
+
+      - name: Cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.pip-cache.outputs.dir }}
+          key: release-${{ hashFiles('**/setup.py') }}
```

Comment:
@hugovk Should we add a `v1` here as well?

Reply:
Could do, it's also trivial to add it later when actually needing to invalidate the old cache to generate a new one.

---

Have you heard about https://github.com/asottile/all-repos? Very good for making lots of (small) changes to many repos. https://jugmac00.github.io/blog/how-to-create-hundreds-of-pull-requests-with-a-single-command/ is a good intro.

Reply:
Yeah, I've seen all-repos, think that'll be excellent to update the workflows in the future, but the actual migration from travis.yml to GHA workflow wasn't something I was able to figure out with it.